x-pack/plugins/security_solution/scripts/run_cypress/parallel.ts

```diff
@@ -54,8 +54,6 @@ export const cli = () => {
         )
         .boolean('inspect');
 
-      const USE_CHROME_BETA = process.env.USE_CHROME_BETA?.match(/(1|true)/i);
```

Comment:
Isn't an ability to test in Chrome beta version required anymore?

Reply:
oh.... no no. this is mistake. I did not meant to remove this. Will add it back. Thanks for catching it.